src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -779,86 +616,21 @@ private void ScanSyntaxToken(ref TokenInfo info)
 
                 case '$':
                     if (TryScanInterpolatedString(ref info))
```

Comment:
Don't really see the need for these stylistic reformats.

Reply:
so the rule i followed was: if i needed to edit the block/section, then i updated to the current coding guidelines we have.  the .net guidelines on blocks are that htey can be eschewed when code is on a single line, so i followed that as the preferable clean form :)

Reply:
restored.